.travis.yml

```diff
@@ -33,7 +33,7 @@ before_script:
     - composer install --prefer-dist --no-interaction -o
 
 script:
-    - ./vendor/bin/simple-phpunit -v --coverage-clover=coverage.clover
+    - XDEBUG_MODE=coverage ./vendor/bin/simple-phpunit -v --coverage-clover=coverage.clover
```

Comment:
This fix is issued by following error:

```
PHP Fatal error:  Uncaught SebastianBergmann\CodeCoverage\RuntimeException: XDEBUG_MODE=coverage or xdebug.mode=coverage has to be set in /home/travis/build/peter279k/php-imgur-api-client/vendor/bin/.phpunit/phpunit-8.3-0/vendor/phpunit/php-code-coverage/src/Driver/Xdebug.php:50
Stack trace:
....
```

And the reference is available [here](https://github.com/sebastianbergmann/php-code-coverage/issues/834).